The build is failing:

`BearerTokenModule.cs(66,69): error CS0103: The name 'StringComparison' does not exist in the current context [/home/travis/build/unosquare/embedio-extras/src/Unosquare.Labs.EmbedIO.BearerToken/Unosquare.Labs.EmbedIO.BearerToken.csproj]`

--- a/src/Unosquare.Labs.EmbedIO.BearerToken/BearerTokenModule.cs
+++ b/src/Unosquare.Labs.EmbedIO.BearerToken/BearerTokenModule.cs
@@ -1,5 +1,6 @@
 ﻿namespace Unosquare.Labs.EmbedIO.BearerToken
 {
+    using System;
     using Microsoft.IdentityModel.Tokens;
     using System.Threading.Tasks;
     using System.Collections.Generic;

